### PR TITLE
Function Example for Frequency Vectorizer

### PR DESCRIPTION
**Pull request by nassimgha**
Hi, I have added the part related to frequency_vectorizer to the example file.
Thanks!

Addresses #46

### CODE REVIEW

File: docs/example.ipynb [hunk 1]
```diff
@@ -105,15 +105,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**one or two sentences describing what this function does in the context of usage and functionality**"
+    "The frequency_vectorizer calculates the frequency of each word in a list of text documents. This function is useful for transforming text data into a feature matrix, and word frequency matrix.\n",
```
Comment (jy1909) [inline docs/example.ipynb]: Small modification.
```suggestion
    "The `frequency_vectorizer` calculates the frequency of each word in a list of text documents to capture the significance of each word in each document. This function is useful for transforming text data into a feature matrix (word frequency matrix) that is to be used for machine learning.\n",
```

**Review by jy1909 — COMMENTED** [review]
Great work! Just a suggestion for a small change.

File: docs/example.ipynb (hunk 1 above)
Comment (nassimgha) [reply docs/example.ipynb]: Thanks, looks good!

Comment (nassimgha) [reply docs/example.ipynb]: I meant to add your suggestions actually.

**Review by jy1909 — APPROVED** [review]
LGTM

**Comment by MoNorouzi23** [comment]
great

**Pull request merged**